modules/miscutil/lib/inveniocfg.py

```diff
@@ -918,7 +918,7 @@ def cli_cmd_create_apache_conf(conf):
         AliasMatch /sitemap-(.*) %(webdir)s/sitemap-$1
         Alias /robots.txt %(webdir)s/robots.txt
         Alias /favicon.ico %(webdir)s/favicon.ico
-        WSGIDaemonProcess invenio processes=5 threads=1 display-name=%%{GROUP} inactivity-timeout=3600 maximum-requests=10000
+        WSGIDaemonProcess invenio processes=5 threads=1 display-name=%%{GROUP} inactivity-timeout=3600 maximum-requests=10000 %(wsgiuser)s
```

Comment:
Hmm, this doesn't look right. There's `invenio`, but also `%(wsgiuser)s`. Perhaps the latter takes presedence, but the [wiki](https://code.google.com/p/modwsgi/wiki/ConfigurationDirectives#WSGIDaemonProcess) states that there is only one `name`. Note the old date of the document however.